### PR TITLE
Convert dask.dataframe to dask.array when using StandardScaler.fit()

### PR DESCRIPTION
**Pull request by cr458**
Closes to #157

### CODE REVIEW

File: dask_ml/preprocessing/data.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ class StandardScaler(skdata.StandardScaler):
     def fit(self, X, y=None):
         self._reset()
         attributes = OrderedDict()
+        if isinstance(X, dd.DataFrame):
```
Comment (TomAugspurger) [inline dask_ml/preprocessing/data.py]: Could you check for dask or pandas DataFrame's here? Then the learned attributes will always be ndarrays.

Comment (cr458) [reply dask_ml/preprocessing/data.py]: Sounds good, have added a check for pandas DataFrames

**Comment by TomAugspurger** [comment]
Added some release notes. This looks good, merging when the tests pass.

**Comment by TomAugspurger** [comment]
Thanks @cr458!

**Comment by cr458** [comment]
@TomAugspurger my pleasure. Keen to help out where I can as there is a lot of overlap between my research and the tools you are developing.

**Pull request merged**